nixos/modules/system/activation/switch-to-configuration.pl

```diff
@@ -339,21 +340,23 @@ sub handleModifiedUnit {
                 # If this unit is socket-activated, then stop the
                 # socket unit(s) as well, and restart the
                 # socket(s) instead of the service.
-                my $socketActivated = 0;
+                my $socket_activated = 0;
```

Comment:
why is this snake case when all the other variables are camel case?

Reply:
https://metacpan.org/pod/Perl::Critic::Policy::NamingConventions::Capitalization

Reply:
makes sense, but what is `unitsToReload`?

Reply:
There is currently a mix between camel case (not yet refactored) and snake case (already refactored)